wiki/People/Performance_Points_Committee/id.md

```diff
@@ -0,0 +1,24 @@
+---
+stub: true
+tags:
+  - pp committee
+  - komite pp
+---
+
+# Performance Points Committee
+
+**Performance Points Committee** (atau Komite Performance Points) merupakan sebuah panel yang beranggotakan pemain yang berpengalaman dan pengembang permainan yang memfasilitasi dan menyetujui perubahan pada algoritma [performance points](/wiki/Performance_points) dan [star rating](/wiki/Beatmapping/Star_rating). Tujuan dari komite tersebut adalah membantu [pengembang osu!](/wiki/People/The_Team/Developers) dalam percepatan penyelesaian suatu perubahan pada algoritma [performance points](/wiki/Performance_points) dan [star rating](/wiki/Beatmapping/Star_rating) dengan koordinasi komunitas yang lebih besar.
```

Comment:
`**Performance Points Committee** (atau *Komite Performance Points*) merupakan sebuah panel beranggotakan para pemain dan pengembang berpengalaman yang bertugas untuk menggawangi berbagai perubahan pada algoritma perhitungan [performance point](/wiki/Performance_points) dan [star rating](/wiki/Beatmapping/Star_rating). Tujuan dari komite ini adalah untuk membantu para [pengembang osu!](/wiki/People/The_Team/Developers) dalam merumuskan algoritma perhitungan [performance point](/wiki/Performance_points) dan [star rating](/wiki/Beatmapping/Star_rating) yang akurat dengan mempertimbangkan berbagai masukan dari komunitas.`

Reply:
>**Performance Points Committee** (atau *Komite Performance Points*) merupakan sebuah panel beranggotakan para pemain dan pengembang berpengalaman yang bertugas untuk menggawangi berbagai perubahan pada algoritma perhitungan [performance point](/wiki/Performance_points) dan [star rating](/wiki/Beatmapping/Star_rating).
In the English article, the one who is experienced is the player only so we may go with `para pengembang dan pemain yang berpengalaman`. `menggawangi` isn't common and formal word, The last sentence is better wording 👍 .